app/metrics.yaml

```diff
@@ -382,6 +382,129 @@ search.default_engine:
     - fenix-core@mozilla.com
     expires: "2019-09-01"
 
+bookmarks_management:
+  bookmark_in_new:
+    type: event
+    description: >
+      A user opened a bookmark in a new tab.
+    bugs:
+    - 974
+    data_reviews:
+    - https://github.com/mozilla-mobile/fenix/issues/974
+    notification_emails:
+    - fenix-core@mozilla.com
+    expires: "2020-04-01"
```

Comment:
for the MVP we're making all the initial events expire in 9 months (March 1st 2020). But going post-MVP we will probably make 6 months the standard.

Reply:
I'll move it to March then.